packages/hoprd/src/index.ts

```diff
@@ -194,10 +194,7 @@ async function generateNodeOptions(environment: ResolvedEnvironment): Promise<Ho
 }
 
 function addUnhandledPromiseRejectionHandler() {
-  process.on('unhandledRejection', (reason: any, promise: Promise<any>) => {
-    console.error('Unhandled Rejection at:', promise, 'reason:', reason)
-    process.exit(1)
-  })
+  require('trace-unhandled/register')
```

Comment:
Now we are missing the exit. Is that somehow configurable?

Reply:
Unfortunately I don't think it is configurable in the package. Node will continue executing, so if termination is what we want at least in production, we need to keep the orignal pattern.

Reply:
I guess this is now fixed with the custom `setLogger`?

Reply:
Correct. Although I'd like to point out their advice:
````
This package is not intended to be used in production, only to aid locating bugs
````
So do we really want to push this to prod?

Reply:
That statement makes sense, you wouldn't want to use this in prod normally, in our case, because we are swamped with these issues, I think we can make the exception. Unless this library doesn't send data to an unknown external server and it purely just adds logs, I see no issue.